process/mock/loggerStub.go

```diff
@@ -0,0 +1,96 @@
+package mock
```

Comment:
I think that a logger mock and/or stub should be exported from the `mx-chain-logger-go`. similar to http vs http test:
```
	"net/http"
	"net/http/httptest"
```

Reply:
Mocks & other test components that have some logic inside.... yes, we might import them but that might prove a careless move in the long run, so I would avoid them